common/src/main/java/com/bakdata/quick/common/api/client/ResponseWrapper.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *    Copyright 2022 bakdata GmbH
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.bakdata.quick.common.api.client;
+
+import com.bakdata.quick.common.exception.MirrorException;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import io.micronaut.http.HttpStatus;
+import lombok.Value;
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+/**
+ * A wrapper for a response from a call to be made with the HttpClient.
+ *
+ * <p>
+ * It consists of a response body extracted from the response and a boolean that indicates
+ * whether the X-Cache-Update header has been set. This header signals the need to update
+ * the mapping between partitions and mirror hosts.
+ * </p>
```

Comment:
```suggestion
 *
```

JavaDoc is weird, there is actually no need to close the tags